react-state-2_table-reservation/src/components/Counter.jsx

```diff
@@ -1,9 +1,6 @@
-import { useState } from "react";
 import "./Counter.css";
 
-
-export default function Counter() {
-  const [people, setPeople] = useState(0);
+export default function Counter(people, onIncrement, onReduce) {
```

Comment:
should look like `Counter({onIncrement, onReduce})`

(people not required and missing {} brackets)

Reply:
Thanks!